RoddyCore/test/de/dkfz/roddy/core/ExecutionContextTest.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.core
+
+import de.dkfz.roddy.RunMode
+import de.dkfz.roddy.execution.io.ExecutionService
+import de.dkfz.roddy.execution.io.LocalExecutionService
+import de.dkfz.roddy.execution.io.fs.FileSystemAccessProvider
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by heinold on 01.02.17.
+ */
+public class ExecutionContextTest {
```

Comment:
No, I put them there, because the checkWorkflowExecutability method is also here. These methods are directly connected to the mentioned method.